CONTRIBUTING.md

```diff
@@ -140,10 +140,26 @@ The content of the commit message body should contain:
 
 Test your code before committing with `% yarn test` or `% yarn packages:test`. It will run type linting, code listing, testing, and code formatting.
 
-## Code style
+## Code Style
 
 This project uses Prettier for code formatting. You can run `make format` to format all your code before you submit your code for review.
 
+## Documenting the Components
```

Comment:
For this section I suggest using [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt): The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.